envs/conda-env-sklearn.yml

```diff
@@ -0,0 +1,27 @@
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.10
```

Comment:
I know pyproject.toml it includes a range of versions. Is it required to hardcode 3.10 here, or is it the pyproject.toml incorrect?

Reply:
Range of versions was changed to [3.9, 3.11], python requirement is removed from conda env file to avoid python reinstallation.